packages/xo-web/src/xo-app/home/index.js

```diff
@@ -1119,6 +1120,15 @@ export default class Home extends Component {
     const nItems = this._getNumberOfItems()
 
     if (nItems < 1) {
+      if (isAdmin) {
```

Comment:
I think it would be even better we isolate the check directly in `NoObjects`:
- `NoObjectsWithoutServers`
- `NoObjectsWithServers`
- `NoObjects` which would subscribe to the store and use one or the other whether the current user is admin.